components/match2/wikis/rocketleague/match_group_input_custom.lua

```diff
@@ -188,6 +188,9 @@ function matchFunctions.getExtraData(match)
 		and opponent1.type == Opponent.team
 		and opponent2.type == Opponent.team
 
+	local hasopponents = Logic.isNotEmpty(opponent1.name) and opponent1.type ~= 'literal'
```

Comment:
`'literal'` should be `Opponent.literal` (same in line below)

do not see the necessity to define the var here and then just use it in extradata, could just define it directly in extradata